diff --git a/src/AppBundle/Form/ProductType.php b/src/AppBundle/Form/ProductType.php
index ab42d53..e43a1d7 100644
--- a/src/AppBundle/Form/ProductType.php
+++ b/src/AppBundle/Form/ProductType.php
@@ -16,8 +16,17 @@ class ProductType extends AbstractType
                     'Yes' => 'stock_yes',
                     'No' => 'stock_no',
                     'Backordered' => 'stock_backordered',
+                    'yes_later' => 'stock_yes_second',
                 ],
                 'choices_as_values' => true,
+                'choice_label' => function($choiceVal, $choiceKey, $choiceIndex) {
+                    if ($choiceKey == 'yes_later') {
+                        // we're purposefully trying to have *two* yes'es
+                        return 'Yes';
+                    }
+
+                    return $choiceKey;
+                },
             ]);
     }
 }
